kernel/configs/fragments/4.19.x/common/serial.conf

```diff
@@ -0,0 +1,14 @@
+# We need some sort of 'serial' for virtio-serial consoles - at the moment.
+# We might not need all of thse though...
+# FIXME - review
+CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_PCI=y
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_CORE_CONSOLE=y
+CONFIG_SERIAL_CORE=y
+CONFIG_SERIAL_EARLYCON=y
+
+# SERIO may be only for keyboards, mice etc., and not UARTS
+# maybe we don't need this?
+CONFIG_SERIO_RAW=y
```

Comment:
I agree, this looks like something that we can get rid of, we dont really need raw access to serial ports.